kingpin/actors/packagecloud.py

```diff
@@ -0,0 +1,256 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2014 Nextdoor.com, Inc
+
+"""
+:mod:`kingpin.actors.packagecloud`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The packagecloud actor allows you to delete packages from a packagecloud repo
```

Comment:
This is is generic high level module documentation, its not yet specific to a given actor. You're documentation here sounds like `packagecloud` is an actor itself, but its not. I would re-phrase this to talk about the _Packagecloud.io actors_, and how they are generally used and set up. From there, the individual classes can have their own usage documentation.